api/src/main/java/io/minio/messages/QueueConfiguration.java

```diff
@@ -97,14 +97,36 @@ public void setEvents(List<EventType> events) {
   /**
    * Returns filter.
    */
+  public Filter getFilter() {
```

Comment:
`get()` style is not used i.e. you would need to just call `filter()`

Reply:
done